suite-common/suite-config/src/fiat.ts

```diff
@@ -45,6 +45,7 @@ export const fiatCurrencies = {
     xdr: { label: 'xdr', value: 'XDR' },
     xag: { label: 'xag', value: 'XAG' },
     xau: { label: 'xau', value: 'XAU' },
+    sol: { label: 'sol', value: 'Solana' },
```

Comment:
Why are you adding solana to fiat? 🤔

Reply:
Check line 55 in this file, the previous object is deprecated.